frame/support/procedural/src/pallet/parse/storage.rs

```diff
@@ -393,10 +404,20 @@ fn process_named_generics(
 			}
 		},
 		StorageKind::CountedMap => {
+			let mandatory_generics: &[&str];
+			let optional_generics: &[&str];
+			if dev_mode {
+				mandatory_generics = &["Key", "Value"];
+				optional_generics = &["Hasher", "QueryKind", "OnEmpty", "MaxValues"];
+			} else {
+				mandatory_generics = &["Hasher", "Key", "Value"];
+				optional_generics = &["QueryKind", "OnEmpty", "MaxValues"];
+			}
```

Comment:
```suggestion
			let mut mandatory_generics = vec!["Key", "Value"];
			let mut optional_generics = vec!["QueryKind", "OnEmpty", "MaxValues"];

			if dev_mode {
				optional_generics.push("Hasher");
			} else {
				mandatory_generics.push("Hasher");
			}
```
Less repetition -> less typos :sparkles:  
(and then use them with a `&` below)